### PR TITLE
Get BeBuilder to compile on Haiku

### PR DESCRIPTION
**Pull request by BachToTheFuture**
This PR comes with a Haiku makefile and a modified README.
Has been tested on x86_64.

### CODE REVIEW

**Comment by scottmc** [comment]
Does this pull request make the other open pull request obsolete?

File: V0.5.2BeBuilder/Makefile [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+		./CodeGenerator/MenuItemCodeGenerator.cpp \
+		./CodeGenerator/MenuSeparatorCodeGenerator.cpp \
+		./Component/MenuItemComponent.cpp \
+		./Component/MenuSeparatorComponent.cpp \
+		./Component/MenuFieldComponent.cpp \
+		./CodeGenerator/MenuFieldCodeGenerator.cpp \
+		./Component/BoxComponent.cpp \
+		./CodeGenerator/BoxCodeGenerator.cpp \
+		./Component/SliderComponent.cpp \
+		./CodeGenerator/SliderCodeGenerator.cpp
```
Comment (owenca) [inline V0.5.2BeBuilder/Makefile]: All the `./` prefixes are redundant.

File: V0.5.2BeBuilder/Makefile [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+		./Component/MenuFieldComponent.cpp \
+		./CodeGenerator/MenuFieldCodeGenerator.cpp \
+		./Component/BoxComponent.cpp \
+		./CodeGenerator/BoxCodeGenerator.cpp \
+		./Component/SliderComponent.cpp \
+		./CodeGenerator/SliderCodeGenerator.cpp 
+
+RDEFS = bebuilder.rdef \
+		./Resource/ViewResource.rdef \
+		./Resources/WindowResource.rdef
```
Comment (owenca) [inline V0.5.2BeBuilder/Makefile]: Same here. And they can fit in one line, right?

File: V0.5.2BeBuilder/Makefile [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+#		$(STDCPPLIBS) instead of the raw "stdc++[.r4] [supc++]" library names.
+#
+#	- 	if your library does not follow the standard library naming scheme,
+#		you need to specify the path to the library and it's name.
+#		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
+LIBS =  be \
+		translation \
+		$(STDCPPLIBS) \
+		tracker \
+		root
```
Comment (owenca) [inline V0.5.2BeBuilder/Makefile]: I would put all the libraries on the same line, and you don't need `root`.

**Review by owenca — CHANGES_REQUESTED** [review]

**Comment by BachToTheFuture** [comment]
@scottmc I believe it does make the other one obsolete: the other PR still has a BeOS Makefile.
@owenca Thanks for reviewing! Will get those fixes committed in a moment.

**Comment by BachToTheFuture** [comment]
@owenca Applied the changes :)

**Comment by scottmc** [comment]
This working now?  How about a refresh of the readme with a screenshot?

**Comment by BachToTheFuture** [comment]
@scottmc: The changes were only cosmetic changes, so I don't think a new screenshot would be necessary :)

**Pull request merged**